identity/identity_filter.go

```diff
@@ -0,0 +1,81 @@
+package identity
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strings"
+
+	"github.com/tidwall/gjson"
+)
+
+func (i Identities) Filter(values url.Values) (Identities, error) {
```

Comment:
This is a nice idea! Unfortunately I think it will become quite difficult once you hit ~10.000 identities or more as you will need to load everything into memory. Another alternative would be to support search for databases that have JSON indexing (so this would not work for SQLite but it does work for PostgreSQL, MySQL 8+, CockroachDB). For SQLite we could use the fallback defined here! Alternative would be to load a JSON extension for SQLite although I don't know how that works.

Reply:
Yes it what I did first by using scope from pop but effectively tests does not work due to SQLite incompatibility

Reply:
I’ll dive in it

Reply:
Just push my previous implementation
give me you thought